ezflow/data/dataset/base_dataset.py

```diff
@@ -46,8 +47,10 @@ def __init__(
             "color_aug_params": {"aug_prob": 0.2},
             "eraser_aug_params": {"aug_prob": 0.5},
             "spatial_aug_params": {"aug_prob": 0.8},
+            "affine_params": {"aug_prob": 0.8},
         },
         sparse_transform=False,
+        normalize=False,
```

Comment:
It will be better to provide this feature as user configurable. 

For example: normalize={"use": True, mean=[0,0,0], std=[255.0,255.0,255.0]}

Reply:
Done